docs/prepare-your-dev-environment.md

```diff
@@ -16,6 +16,8 @@ This document goes through the development dependencies one requires in order to
 
 1. Install the relevant packages required for your OS defined below.
 
+1. Install [Docker 17.05+](https://docs.docker.com/engine/install/fedora/) or later if you haven't already, used for building container images.
```

Comment:
This documentation is specific to Fedora.  Additionally come late Jan, the docker desktop license requires a subscription on MacOS.  I would suggest that we move this line to the "fedora packages" portion as well as maybe reference podman instead of docker (although the docker engine is not in the subscription requirement).  

Additionally, there's a package called `podman-docker` which creates a simple shell script in `/usr/bin/docker` on Fedora which just execs the `podman` binary.  I would suggest adding these steps to the Fedora package section which would be more development environment specific.  

Summarizing + steps to add
1. Under fedora packages, add install `podman` and `podman-docker`
2. Touch a file at `/etc/containers/nodocker` for `az acr login` compatability
3. Add (or ask someone on Mac) the appropriate steps for doing this on MacOS

Reply:
Need to add the steps appropriate for MaC, will check with someone on Mac and update.